bench/vector_tile.js

```diff
@@ -56,9 +56,9 @@ Tile.Feature.read = function (pbf, end) {
 };
 Tile.Feature._readField = function (tag, obj, pbf) {
     if (tag === 1) obj.id = pbf.readVarint();
-    else if (tag === 2) obj.tags = pbf.readPackedVarint();
+    else if (tag === 2) obj.tags = pbf.readPackedVarint(obj.tags);
```

Comment:
Yeah, I think we're fine not to keep this kind of compatibility. However, it's probably a good idea to include `pbf` version in a comment in a generated file to indicate compatibility requirements.